main.go

```diff
@@ -93,6 +93,11 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
```

Comment:
In think this link explains it even better: https://www.postgresql.org/docs/9.3/static/mvcc.html The algorithm used requires serializable transactions